adapters/repos/db/backup.go

```diff
@@ -66,10 +66,7 @@ func (db *DB) BackupDescriptors(ctx context.Context, bakid string, classes []str
 				desc.Error = fmt.Errorf("class %v doesn't exist any more", c)
 			} else if err := idx.descriptor(ctx, bakid, &desc); err != nil {
 				desc.Error = fmt.Errorf("backup class %v descriptor: %w", c, err)
-			} else {
-				desc.Error = ctx.Err()
```

Comment:
Good question: 
With the `descriptor()` now handling  CTX, this function will release resources, if context is cancelled. The caller of `BackupDescriptors()` relies on it either return a valid result or, in case of error to simply return the error and handle the release of resources.